manifest.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: 1
+applications:
+  - name: s3-interface
+    routes:
+      - route: identity-idva-usps-((ENVIRONMENT)).apps.internal
+    memory: ((MEMORY))
+    instances: ((INSTANCES))
+    buildpacks:
+      - python_buildpack
+    command: gunicorn s3.main:app
+    no-route: true
+    health-check-type: process
```

Comment:
if you are listening on 8080, you can leave default heath check type, don't need to change it to process